jekyll/index.html

```diff
@@ -313,23 +231,3 @@ <h2>Twitter</h2>
 
   </section>
 </section>
-
-<section class="content center-banner">
-  <h1 class="section-heading">
-    <i class="fab fa-github fa-2x" aria-hidden="true"></i>
-    Looking for the GitHub repository?
-  </h1>
-  <div class="pure-g center">
-    <div class="pure-u-1-2">
-      <h3>
-        The Nim compiler and tools are all written in Nim and licensed under the
-        MIT license, with most development taking place on GitHub.
-        Be sure to watch the repository to get updates on Nim's
-        development, or star it to give us some brownie points.
-      </h3>
-    </div>
-    <div class="pure-u-1 center">
-      <a class="pure-button" href="https://github.com/nim-lang/Nim">Source code</a>
-    </div>
-  </div>
-</section>
```

Comment:
Personally I think this is what most visitor's look for, which is why I specifically added it to the website. It's at the bottom of the page anyway, why remove it?

I have also done my best to make sure there is a white panel between the footer and the community panel. I see you've worked around this by adding a border on the footer, but IMO that degrades the aesthetics of the site.

Reply:
> It's at the bottom of the page anyway, why remove it?

Because my boss told me to do so ;) and I don't have any convincing arguments against its removal. There is a "Source" link in the top bar which serves the purpose.

> I see you've worked around this by adding a border on the footer

It was always there, I just changed it to look slightly better.

Reply:
In that case @Araq :)